packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -42,20 +44,8 @@ export type FeedItemComponentProps = {
   user: LoggedUser | undefined;
   feedName: string;
   ranking?: string;
-  onUpvote: (
-    post: Post,
-    index: number,
-    row: number,
-    column: number,
-    upvoted: boolean,
-  ) => Promise<void>;
-  onDownvote: (
-    post: Post,
-    index: number,
-    row: number,
-    column: number,
-    upvoted: boolean,
-  ) => Promise<void>;
+  onUpvote: UseVotePost['toggleUpvote'];
```

Comment:
Just wondering would this not be better:
`Pick<UseVotePost, 'toggleUpvote'>;`

Reply:
By picking you would get `{ toggleUpvote }` right?

We could do `FeedItemComponentProps & <Pick<UseVotePost, 'toggleUpvote'>`. Did you think that?

Reply:
Yes that inded would work right?

Reply:
Sure, I'll update.

Reply:
8d638aaa796be50d41b6a60aebd3c3a5511a1e61